### ISSUE TITLE
Not shows layers except base layer.

### ISSUE THREAD

**Issue by petunjiaj** [3liz/lizmap-web-client#1194]
Hi to all,
I installed Lizmap on a Raspberry pi (local network working).
Everything goes right, but after two clean installations, I have always same problem:
I can't see layers on maps.
For example: in demo project montpelier (but for any project) I can see only base layer.

Note: With raspberry I can use Qgis-server 2.14 (<=2.14)
          Lizmap Version number 3.3pre.181017

tank you.

(i read [this post](https://github.com/3liz/lizmap-web-client/issues/632), but it seems to not be very usefull to understand)

**Comment by rufuscorp** [comment]
Something in the lizmap's log ?

**Comment by petunjiaj** [comment]
> 2019-04-11 15:34:23	192.168.1.74	warning	2019-04-11 15:34:23	[2]	SQLite3::query(): Unable to prepare statement: 1, no such table: lizmap_search	/var/www/mylizmap/lib/jelix/plugins/db/sqlite3/sqlite3.dbconnection.php	112

but I think is nothing important for this issue.

**Comment by rufuscorp** [comment]
I can see the same error with:

`no such table: lizmap_search`

lizmap 3.2.2 on debian stretch freshly install, with qgis server 3.4.

Only base layer with Montpellier appear, and no layer with another qgis project . I can't see the data too.
I've install the 3.1.12 on the same server, and it works: i can see layer and data.

It seem there's a problem with the 3.2.2 version.

**Comment by petunjiaj** [comment]
I resolved my problem.

When I installed Qgis-server I added the example project as tutorials suggest 
(Trainer demo data ->project World, see [here](https://docs.qgis.org/3.4/en/docs/training_manual/qgis_server/wms.html))
Doing that I changed the qgis.conf (or default-000.conf by user choice) site configuration file, in apache sites-available.
As suggested I added the default project path under the relative line: 
`# default QGIS project`
`    SetEnv QGIS_PROJECT_FILE /home/qgis/projects/world/world.qgs` 

Doing that I wasn't able to load layers on Lizmap projects.
SOLUTION: comment or remove default project in qgis site configuration file.
note: after that restart apache service.

NOTE:
 I presumed it was not a Lizmap problem when I wasn't able to load demo project by WMS (with qgis).
Whenever I tried to connect by wms server I was able to find only world-project (no lizmap demo project or other lizmap qgis projects).
So I tried this solution.

Hope this can help.
